lib/all_your_base_cases.rb

```diff
@@ -57,16 +57,17 @@ def error_message
 
 class AllYourBaseCase::PreProcessor
   class << self
+    attr_reader :row
+
     def call(row)
       @row = row
 
       row.merge('expected' => expected_value)
     end
 
+    private :row
     private
 
```

Comment:
Yeah, that's what I had originally but it's what causes the warning mentioned above. Ruby versions less than 2.3.0 have a warning when a private attribute is defined unless its made private explicitly with private :method_name